content/docs/06-integrations/00-byoos.md

```diff
@@ -0,0 +1,239 @@
+
+
+- Palette 3.3.0 or greater.
+
+## Parameters
+
+The BYOS Edge OS pack supports the following parameters. 
+
+### Parameters
+
+| Parameter            | Description                                            | Type |
+|----------------------|--------------------------------------------------------|------|
+| `pack:content:` | Specifies the content of the **BYOS Edge OS** pack. | map |
+| `pack.content.images` | Specifies a list of OS images to use with the pack. | list |
+| `pack.content.images.image` | An OS image to use with the pack. | string|
+| `system.uri` | The system URI specifies the BYOOS image's location. | string|  
+
+
+
+  ```yaml
+  pack:
+  content:
+    images: 
+      - image: '{{.spectro.pack.edge-native-byoi.options.system.uri}}'
+      # - image: example.io/my-other-images/example:v1.0.0 
+      # - image: example.io/my-super-other-images/example:v1.0.0 
+    
+  options: 
+    system.uri: example.io/my-images/example-custom-os:v1.4.5
+  ```
+
+## Usage
+
+BYOOS enables you to use a custom OS for your Edge host. You can use this feature to customize the desired specifications of your OS layer in the Edge host. You can reference the custom OS through the BYOOS pack.
+
+
+To use a custom OS, you must include all the Edge artifacts and provider images required by the Edge Installer in the custom OS. Refer to the [Bring Your Own OS](/clusters/edge/edgeforge-workflow/build-kairos-os) guide for steps on how to create a custom OS that includes all the required components for the Edge Installer.
+
+
+Select the BYOOS pack and fill out the required parameters during the cluster profile creation process. The `system.uri` parameter specifies the location of the BYOOS image. Refer to the [Build Images](/clusters/edge/edgeforge-workflow/build-images) guide to learn how to create Edge Artifacts.
+
+![A view of the Kubernetes pack editor with a YAML configuration](/clusters_site-deployment_model-profile_byoos-pack-yaml.png)
+
+
+</Tabs.TabPane>
+
+<Tabs.TabPane tab="Non-Edge" key="Non-Edge">
+
+## Prerequisites 
+
+To use the non-Edge BYOOS pack, you must have the following:
+
+<br />
+
+- A custom OS you created is available. Refer to the [Create Images with Image Builder](/cluster-profiles/byoos/image-builder) guide to learn how to create a custom OS for Palette.
+
+## Parameters
+
+The following is a list of parameters required when using the BYOOS pack.
+
+<br/>
+
+| Parameter            | Description                                            | Type |
+|----------------------|--------------------------------------------------------|---|
+| `osImageOverride` | The image ID used as the base OS layer. This is the image ID as assigned in the infrastructure environment it belongs to. Example: `ami-0f4804aff4cf9c5a2` | string|
+| `osName` | The name of the OS distribution. Example: `rhel`. | string |
```

Comment:
```suggestion
| `osName` | The name of the OS distribution. Example: `rhel` | string |
```

Reply:
Following examples like these, it makes sense to omit the period, which can be distracting or confused as being part of the parameter.